deployments/apiv2/services/users/src/main.py

```diff
@@ -305,11 +306,12 @@ async def register(
             phash = ph.hash(details.password1.get_secret_value())
             scope = details.scope
             # usage_restrictions column is not currently being inserted into, will need to be manually added
-            insert_query = "INSERT INTO customers (email, password, data) VALUES ($1, $2, $3) RETURNING id"
+            insert_query = "INSERT INTO customers (email, password, data, usage_restrictions) VALUES ($1, $2, $3, $4) RETURNING id"
             query_params = (
                 details.email,
                 phash,
                 json.dumps({"scope": scope}),
+                json.dumps(PULSE3D_PAID_USAGE),
```

Comment:
It doesn't look like `json.dumps` likes `immutabledicts`, but `json.dumps(dict(PULSE3D_PAID_USAGE))` should work

Reply:
@tannermpeterson thanks for catching these, my bad for missing them. Christmas brain.